irohad/torii/impl/command_service.cpp

```diff
@@ -90,8 +87,8 @@ namespace torii {
               cache_->addItem(tx_hash, response);
               // Send transaction to iroha
               tx_processor_->transactionHandle(
-                  std::shared_ptr<iroha::model::Transaction>(
-                      iroha_tx.value.makeOldModel()));
+                  std::make_shared<shared_model::proto::Transaction>(
+                      iroha_tx.value));
```

Comment:
`std::move`, so that no copy is made. Requires non-const parameter.

Reply:
Not possible at the moment due to perfect forwarding constructor